components/ConnectWalletButton.js

```diff
@@ -0,0 +1,10 @@
+// components/ConnectWalletButton.js
+import { ConnectButton } from "@rainbow-me/rainbowkit";
+
+export default function ConnectWalletButton() {
+  return (
+    <div>
```

Comment:
Use Chakra UI components for styling :
https://chakra-ui.com/docs/components/layout/box

The same applies or components in Header.js